certified-connectors/PKIsigning/apiProperties.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "E7A018BA-2E96-4D90-9ADD-CF593DE86CD5",
+          "scopes": [
+            "pkisigning.platform",
+            "ImpersonateOwner",
+            "openid",
+            "profile",
+            "offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://identity.pkisigning.io/connect/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://identity.pkisigning.io/connect/token"
+            },
+            "refreshUrl": {
+              "value": "https://identity.pkisigning.io/connect/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0E6EFE",
+    "capabilities": [],
+    "publisher": "Wim Zoet"
```

Comment:
@pkisigning The publisher name seems an individual name which is not allowed, It should be an organization or company name. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

Reply:
publisher has been updated to be equal to stackOwner property and references our company name. Please note PKIsigning is not our company name, but just our trademark. This is mentioned in readme.md as well.